src/worker/typosquat/config.rs

```diff
@@ -0,0 +1,65 @@
+//! Things that should really be read from a configuration file, but are just hardcoded while we
+//! experiment with this.
+
+use std::time::Duration;
+
+/// How long to cache the top crates for.
+pub(super) static CACHE_TTL: Duration = Duration::from_secs(12 * 60 * 60);
+
+/// Valid characters in crate names.
+pub(super) static CRATE_NAME_ALPHABET: &str =
+    "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz1234567890-_";
+
+/// E-mail addresses to notify when a potential typosquat is found.
+pub(super) static NOTIFY_EMAILS: &[&str] = &[
+    "adamharvey@rustfoundation.org",
+    "walterpearce@rustfoundation.org",
+];
```

Comment:
I think I'd prefer it if we would read these from an env var instead

Reply:
Done; we now read from `$TYPOSQUAT_NOTIFICATION_EMAILS` (feel free to bikeshed).